scripts/time_plot.py

```diff
@@ -1497,6 +1497,69 @@ def handle_websock_event(handlerkey,*args):
                 for thishandler in websockrequest_username.keys():
                     if (websockrequest_username[thishandler]==username):
                         send_websock_cmd('ApplyViewLayout('+'["'+'","'.join([fig['figtype'] for fig in html_viewsettings[username]])+'"]'+','+str(html_layoutsettings[username]['ncols'])+')',thishandler)
+        elif (args[0].startswith('holo')):#accepts 'holo', 'holo 3' where 3 is m003 as explicit reference antenna, else first track antenna is chosen as reference antenna
+            logger.info(repr(args))
```

Comment:
I haven't looked through all the code, but it seems that you're repeating this in each block, instead of just doing it once at the top? Or do you not do it for some commands?

Reply:
There is a lot of repetition but some commands have arguments and others do not. It is possible to simplify but is more effort than it is worth at this stage.